classes/02class/exercises/c02-network08/drii-cavalcanti/c02-network08.txt

```diff
@@ -0,0 +1,15 @@
+1. What is the difference between inbound x outbound?
+   The difference between inboud(ingress) and outbound(egress) is the direction of the traffic based on the point-of-view of the subnets interface. In other words, inbound is the traffic that an instance receives and outbound is the traffic that an instance sends.
```

Comment:
Your definition of inbound and outbound is correct but for this exercise it is about ACL inbound and outbound network rules.

So when you say `inbound is the traffic that an instance receives and outbound is the traffic that an instance sends`, in fact this happens at the `network` level, not `instance` level.  The ACL will block incoming traffic even before reaching an EC2 instance, for example.